### DISCUSSION TITLE
How to invoke non-static method ?

### DISCUSSION THREAD

**Discussion by Mandofskii** [vfsfitvnm/frida-il2cpp-bridge#212] [Q&A]
How to invoke non-static method ?

**Comment by bluewave41** [comment]
By getting an object instance.

https://github.com/vfsfitvnm/frida-il2cpp-bridge/wiki/Snippets#heap-scanning describes how to pull objects off the "heap"

Or you can allocate your own object which is probably not what you want to do using

    const object = Il2Cpp.Domain.tryAssembly('domain').image.class('class').alloc();
    object.method('.ctor').invoke();
    object.method('whatever').invoke();
